### ISSUE TITLE
 Settings. 2012 - Could not save data in elasticsearch due to [cluster_block_exception] blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];

### ISSUE THREAD

**Issue by xisafe** [wazuh/wazuh-kibana-app#1016]
## Bug report

**OS**
CentOS7

**Wazuh version**
3.7

**Wazuh API version**
3.8

Settings. 2012 - Could not save data in elasticsearch due to [cluster_block_exception] blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];

**Comment by jesusgn90** [comment]
Hi @xisafe ,

One or more indices are blocked, so they can't be updated. My bets are `.wazuh` and `.wazuh-version`.

Please execute the next command:

```
curl -XGET "http://localhost:9200/.wazuh,.wazuh-version/_settings?pretty"
```

Replace `localhost` with your Elasticsearch IP. If an index is blocked, you should see this as part of the index information:

```
"blocks" : {
          "read_only_allow_delete" : "true"
        },
```

If some index is blocked, use this command to unlock it:

```
curl -XPUT "http://localhost:9200/.wazuh/_settings" -H 'Content-Type: application/json' -d'
{
  "index": {
    "blocks": {
      "read_only_allow_delete": "false"
    }
  }
}'
```

You can verify it once done using the first command again:

```
curl -XGET "http://localhost:9200/.wazuh,.wazuh-version/_settings?pretty"
```

I hope it helps.

Best regards,
Jesús

**Comment by jesusgn90** [comment]
@xisafe  assuming this is solved for you, re-open this ticket or open a new ticket if you are still facing troubles.

Regards!
